### PR TITLE
Noibu Error Tracking Trace Added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Request

**`Examples:`**

https://www.sharperimage.com/
https://www.onehanesplace.com/brands/just-my-size.html
https://iheartcats.com/
https://www.signaturehardware.com/
https://www.silhouettedesignstore.com/

### CODE REVIEW

File: src/technologies/n.json [hunk 1]
```diff
@@ -872,6 +872,17 @@
     "saas": true,
     "website": "https://nodeping.com"
   },
+  "Noibu": {
+    "cats": [
+      13
+    ],
+    "description": "Noibu is used to deliver better digital experience by preventing lost of revenue. Detect, prioritize & resolve critical eCommerce errors.",
+    "icon": "noibu.png",
```
Comment (nurbek91) [inline src/technologies/n.json:880]: Noibu
svg available

File: src/technologies/n.json [hunk 2]
```diff
@@ -872,6 +872,17 @@
     "saas": true,
     "website": "https://nodeping.com"
   },
+  "Noibu": {
+    "cats": [
+      13
+    ],
+    "description": "Noibu is used to deliver better digital experience by preventing lost of revenue. Detect, prioritize & resolve critical eCommerce errors.",
```
Comment (nurbek91) [inline src/technologies/n.json:879]: Noibu helps ecommerce websites detect revenue-impacting errors on their websites and provides them with the information needed to resolve them.

File: src/technologies/n.json (hunk 1 above)
Comment (nurbek91) [reply src/technologies/n.json:880]:
```
"pricing": [
      "poa"
    ],
"saas": true,
```

File: src/technologies/n.json [hunk 3]
```diff
@@ -872,6 +872,17 @@
...
+    "icon": "noibu.png",
+    "js": {
+      "NOIBUJS_CONFIG": ""
```
Comment (nurbek91) [inline src/technologies/n.json:882]: NOIBUJS_CONFIG.scriptID Isn't that the version by any chance? I'm not sure

**Review by nurbek91 — CHANGES_REQUESTED** [review]

File: src/technologies/n.json (hunk 3 above)
Comment (dhavalveera) [reply src/technologies/n.json:882]: nope, it's not a version, because everyone have the scriptID, any I don't think that it'll be a version

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**